e2e/resources/workbench-config.ts

```diff
@@ -4,18 +4,18 @@ const _ = require('lodash');
 const env = process.env.WORKBENCH_ENV || 'dev';
 
 const userCredential = {
-  registrationContactEmail: process.env.CONTACT_EMAIL,
```

Comment:
contact_email is not a "secret", so I'm taking it out of `.env`. hardcode in `workbench-configs` is okay.